### PR TITLE
Update yeelight.markdown

### PR DESCRIPTION
**Pull request by ApplY3D**
## Proposed change
Update information about enabling LAN mode for Xiaomi bulbs. 
The old text is just been strikethrough, maybe there is a better way to mark the official guide website as deprecated.

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Checklist
- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
  - Updated information on enabling "LAN Control" for Yeelight bulbs, noting the absence of an official method and suggesting alternative approaches.

### CODE REVIEW

File: source/_integrations/yeelight.markdown [hunk 1]
```diff
@@ -161,7 +161,10 @@ Per default, the bulb limits the number of requests per minute to 60, a limitati
 Before trying to control your light through Home Assistant, you have to set up your bulb using the Yeelight app. ( [Android](https://play.google.com/store/apps/details?id=com.yeelight.cherry&hl=fr), [IOS](https://itunes.apple.com/us/app/yeelight/id977125608?mt=8) ).
 In the bulb property, you have to enable "LAN Control" (previously called "Developer mode"). LAN Control may only be available with the latest firmware installed on your bulb.  Firmware can be updated in the application after connecting the bulb.
 Determine your bulb IP (using router, software, ping...).
-Information on how to enable "LAN Control" can be found [here](https://www.yeelight.com/faqs/lan_control).
+~~Information on how to enable "LAN Control" can be found [here](https://www.yeelight.com/faqs/lan_control).~~
```
Comment (c0ffeeca7) [inline source/_integrations/yeelight.markdown]: Could you provide a bit more context? I do not quite understand why you say there "is no official way to change LAN mode". The linked yeelight instructions seem to show how to do this.
image

Comment (ApplY3D) [reply source/_integrations/yeelight.markdown]: https://www.reddit.com/r/homeassistant/comments/mfdlva/yeelight_no_more_lan_mode/

Comment (c0ffeeca7) [inline source/_integrations/yeelight.markdown]:
```suggestion
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @ApplY3D :+1:

**Pull request merged**